pages/general-information-and-resources/tech-policies/tts-pages.md

```diff
@@ -0,0 +1,84 @@
+---
+title: TTS Pages
+questions:
+  - tts-tech-operations
+  - cg-pages
+redirect_from:
+  - /tts-pages/
+---
+# TTS Pages - Authority to Use (ATU) Process
+
+>This guide is intended for `Website Managers` to meet their requirements under the TTS Pages Authority to Operate (ATO)
```

Comment:
I suggest something other than blockquote for notes like this. If they need to be set off from the rest of the text under the heading, maybe just give them something like a "Note: " prefix or something else that indicates how they are distinct. If they need different markup I'd probably choose italics.

Reply:
Switching to alert component